### ISSUE TITLE
List as body parameter fails in validation

### ISSUE THREAD

**Issue by cansik** [zalando/connexion#66]
As I saw you modified body parameter handling in version 0.10.2 #59. Now the problem is that lists cause an error because you try to call `items` on a `list` object which fails.

It looks like you just handle dicts (JSON Objects) as body parameters ([code](https://github.com/zalando/connexion/blob/9593e9e4611d9d434f00d7057b4e5f2de4308273/connexion/decorators/parameter.py#L57)):

```python
        # Add body parameters
        for key, value in body_parameters.items():
            if key not in arguments:
                logger.debug("Body parameter '%s' not in function arguments", key)
```

My swagger config for this route looks like this:

```yaml
  /provenance/:
    post:
      tags:
        - INGEST
      description: creates a new provenance if it does not exist
      operationId: propertyservice.routes.provenance.create
      parameters:
        - name: provenance_data
          in: body
          description: provenance_data
          required: true
          schema:
            type: array
            items:
              type: string
      responses:
        200:
          description: OK
```

And this is what I send with the POST request as body:

```json
["test2"]
```

#### Full error message
```python
INFO:werkzeug:127.0.0.1 - - [25/Sep/2015 13:52:47] "POST /provenance/ HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/Library/Python/2.7/site-packages/flask_cors/extension.py", line 110, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Python/2.7/site-packages/flask_cors/extension.py", line 110, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Library/Python/2.7/site-packages/connexion/decorators/validation.py", line 160, in wrapper
    response = function(*args, **kwargs)
  File "/Library/Python/2.7/site-packages/connexion/decorators/validation.py", line 271, in wrapper
    response = function(*args, **kwargs)
  File "/Library/Python/2.7/site-packages/connexion/decorators/produces.py", line 120, in wrapper
    data, status_code = self.get_data_status_code(function(*args, **kwargs))
  File "/Library/Python/2.7/site-packages/connexion/decorators/parameter.py", line 57, in wrapper
    for key, value in body_parameters.items():
AttributeError: 'list' object has no attribute 'items'
```

**Comment by hjacobs** [comment]
Yes, looks like a bug :beetle: at first glance --- we will check this.
Thanks for reporting :smile:

**Comment by jmcs** [comment]
Fixed by #68
